km_api/know_me/models.py

```diff
@@ -664,14 +664,10 @@ class ProfileTopic(mixins.IsAuthenticatedMixin, models.Model):
         name (str):
             The topic's name.
     """
-    GROUPED = 1
-    PAGED = 2
-    TEXT = 3
-    VISUAL = 4
+    TEXT = 1
```

Comment:
Cool, I wasn't sure which way to go. So, just to be clear, change it back to `Text=3 Visual=4` , and leave a comment explaining why?